doc/src/modules/integrals/rubi/about.rst

```diff
@@ -0,0 +1,25 @@
+About Rule Based Integrator
+===========================
+
+About it
+--------
+
+A RUle Based Integrator nicknamed RUBI is a module in sympy.integrals. It is an
```

Comment:
RUBI is an external project, we are importing the rules. It sounds a bit weird if you start like this a sentence.